index.html

```diff
@@ -11,69 +11,338 @@
+                            </li>
+                            <li class="tag2">
+                                <h6 class="tag-name2">CSS</h6>
+                            </li>
+                            
+                            <li class="tag3">
+                                <h6 class="tag-name3">JavaScript</h6>
+                            </li>
+                            <li class="tag4">
+                                <h6 class="tag-name4">html</h6>
+                            </li>
+                                    </ul>
+                                </div>
+                            </div>
+
+                            <div class="actions">
+                                <div class="btn">
+                                    <h6 class="btn-text">See Project</h6>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                </div>
+            </section>
+        <section class="about-me">
+            <div class="information">
+                <h3 class="aboutme-text">
+                    About me
+                </h3>
+                <p class="description-text"> I can help you build a product , feature or website Look through some of my
+                    work and experience! If you like what you
+                    see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="get-mycv">
+                    <h6 class="btn-cv">Get my Resume </h6>
                 </div>
             </div>
-        </div>
-    </div>
-    </div>
+            <div class="card">
+                <div class="language-card">
+                    <div class="lang-placeholder">
+                        <div class="title-tag">
+                            <h3 class="lang-title">Languages</h3>
+                            <ul class="tags">
+                                
+                                    <li class="tag-lang-name1"><h6 class="tag-language-name1"> JavaScript </h6></li>
+                                    <li class="tag-lang-name1"><h6 class="tag-language-name1">Ruby</h6> </li>
+                                    <li class="tag-lang-name1"><h6 class="tag-language-name1">html</h6> </li>
+                                    <li class="tag-lang-name1"><h6 class="tag-language-name1">CSS</h6></li>
+                                
+                            </ul>
+                        </div>
+                        <div class="icons">
+                            <div class="icon-languages">
+                                <div class="ellipse">
+                                    <div class="icon-langguage">
+                                        <div class="frame">
+                                            <img src="Lang.png" alt="language icon"/>
+                                        </div>
+                                    </div>
```

Comment:
- [ ] To enforce proper use of semantics kindly ensure the icon is wrapped with the `a` tag, as this serves as link.